core/lib/scmi-server/sub.mk

```diff
@@ -148,6 +151,20 @@ ifneq (,$$(wildcard $(scpfw-path)/$3/$2/include/*))
 incdirs_ext-y += $(scpfw-path)/$3/$2/include
 endif
 srcs-$(CFG_SCPFW_MOD_$4) += $(scpfw-path)/$3/$2/src/mod_$1.c
+
+# SCMI_Perf has components that can be added conditionally
+ifeq ($(1), scmi_perf)
+
+ifneq (,$(filter y, $(CFG_SCPFW_SCMI_PERF_PROTOCOL_OPS)))
+srcs-$(CFG_SCPFW_MOD_SCMI_PERF) += $(scpfw-path)/$3/$2/src/scmi_perf_protocol_ops.c
+endif
+
+ifneq (,$(filter y, $(CFG_SCPFW_SCMI_PERF_FAST_CHANNELS)))
```

Comment:
Do you foresee added CFG_ dependencies? Unless what, `ifneq ($(CFG_SCPFW_SCMI_PERF_xxx),y)` is simpler.
Ditto at line 159.

Reply:
Not really.. I will change this and push an updated PR. Thanks!